src/gtest/test_foundersreward.cpp

```diff
@@ -107,42 +112,53 @@ void checkNumberOfUniqueAddresses(int nUnique) {
 
 
 TEST(founders_reward_test, general) {
+
     SelectParams(CBaseChainParams::TESTNET);
 
     CChainParams params = Params();
     
-    // Fourth testnet reward:
-    // address = zrBAG3pXCTDq14nivNK9mW8SfwMNcdmMQpb
-    // script.ToString() = OP_HASH160 9990975a435209031e247dccf9bc3e3ed3c81339 OP_EQUAL
-    // HexStr(script) = a9149990975a435209031e247dccf9bc3e3ed3c8133987
-    
-    EXPECT_EQ(params.GetFoundersRewardScriptAtHeight(1), ParseHex("a9149990975a435209031e247dccf9bc3e3ed3c8133987"));
-    EXPECT_EQ(params.GetFoundersRewardAddressAtHeight(1), "zrH8KT8KUcpKKNBu3fjH4hA84jZBCawErqn");
-    EXPECT_EQ(params.GetFoundersRewardScriptAtHeight(53126), ParseHex("a914581dd4277287b64d523f5cd70ccd69f9db384d5387"));
-    EXPECT_EQ(params.GetFoundersRewardAddressAtHeight(53126), "zrBAG3pXCTDq14nivNK9mW8SfwMNcdmMQpb");
-    EXPECT_EQ(params.GetFoundersRewardScriptAtHeight(53127), ParseHex("a914581dd4277287b64d523f5cd70ccd69f9db384d5387"));
-    EXPECT_EQ(params.GetFoundersRewardAddressAtHeight(53127), "zrBAG3pXCTDq14nivNK9mW8SfwMNcdmMQpb");
 
+    /*
+    CBitcoinAddress add ("zrRBQ5heytPMN5nY3ssPf3cG4jocXeD8fm1");
+        CScriptID scriptID = boost::get<CScriptID>(add.Get()); // Get() returns a boost variant
+        CScript script = CScript() << OP_HASH160 << ToByteVector(scriptID) << OP_EQUAL;
+        std::cout<<"Script: "<<script.ToString()<<std::endl;
+        std::cout<<HexStr(script)<<std::endl;*/
```

Comment:
I do not understand why this is here. Could you comment on the purpose of this snippet or remove it?